internal/config/cli.go

```diff
@@ -22,6 +22,13 @@ import (
 	"strings"
 )
 
+const (
+	CUDACompatModeMount    = "mount"
+	CUDACompatModeLdconfig = "ldconfig"
+	CUDACompatModeHook     = "hook"
+	CUDACompatModeDisabled = "disabled"
```

Comment:
So the runtime has one more "mode" than the nvidia-container-cli, i.e. the `hook` mode -- is that the interpretation here?